packages/grafana-ui/src/components/FormLabel/FormLabel.tsx

```diff
@@ -33,10 +36,8 @@ export const FormLabel: FunctionComponent<Props> = ({
     <label className={classes} {...rest} htmlFor={htmlFor}>
       {children}
       {tooltip && (
-        <Tooltip placement="top" content={tooltip} theme={'info'}>
-          <div className="gf-form-help-icon gf-form-help-icon--right-normal">
-            <Icon name="info-circle" size="sm" style={{ marginLeft: '10px' }} />
-          </div>
+        <Tooltip placement="top" content={tooltip} theme={'info'} interactive={interactive}>
+          <Icon tabIndex={0} name="info-circle" size="sm" style={{ marginLeft: '10px' }} />
```

Comment:
I'm not particularly familiar with the `Tooltip` component so I'm unsure what the need for `tabIndex={0}` is here? Is it required for `interactive`? If so should it be set only when the `interactive` prop is set?

Reply:
Ah - great spotted @jackw! Agree with you.

Reply:
I'm addressing two things here so this is confusing, apologies! :grimacing: 

1. `interactive` is needed so that links in tooltips can be clickable otherwise the tooltip is dismissed before the user's cursor can reach the link.
2. `tabIndex` is needed so that the tooltip content is display-able via `Tab`.

Both are needed but solve different issues.